### DISCUSSION TITLE
Installing Nyoon as another Nvim config

### DISCUSSION THREAD

**Discussion by idr4n** [nyoom-engineering/nyoom.nvim#124] [Q&A]
Hi and thanks for a really nice and interesting project. 

I really like Nyoom and I'm looking forward to exploring further it to adopt it as my main Nvim configuration. 

However, it’s going to take time for me to become productive in it and learn to modify it to my own needs. 

I would like to know if there is any recommended way to install Nyoom as an alternative configuration to my current Nvim configuration. For example, to be installed in a different config folder other than `~/.config/nvim/`, and to make sure that it is independent from `~/.local/share/nvim` and `~/.local/state.nvim` folders.

Thanks in advance!

**Comment by mikalcallahan** [comment]
Maybe not the _most effective_ solution but I version control my nvim config with git and created a separate nyoom branch that I switch between as needed

**Comment by TylerBarnes** [comment]
You can use `NVIM_APPNAME`. The install/sync command partially supports it except for one part.

1. clone to wherever you want it as a separate config: `git clone --depth 1 https://github.com/nyoom-engineering/nyoom.nvim.git ~/.config/Nyoom` for ex
2. `cd ~/.config/Nyoom && bin/nyoom install`
3. when you run `bin/nyoom sync` you will get an error about packer not being found
4. edit `bin/nyoom`. on line 80 there's a line where nvim is started with `--headless` but the NVIM_APPNAME isn't set, so it's not starting nvim in the right location. Add `NVIM_APPNAME=$NVIM_APPNAME ` to the beginning of that line.
5. run `NVIM_APPNAME=Nyoom bin/nyoom sync`
6. everything should install fine and you can now run Nyoom as a separate config with `NVIM_APPNAME=Nyoom nvim`

**Reply by TylerBarnes** [reply]
I opened a PR to make this easier here https://github.com/nyoom-engineering/nyoom.nvim/pull/154
